From 4a45e9952dd2985bc07c7210f471aae638c22a6c Mon Sep 17 00:00:00 2001
From: sjorge <sjorge@blackdot.be>
Date: Sat, 2 Jun 2018 10:19:45 +0200
Subject: [PATCH] OS-6984 bhyve bootrom should be configurable via vmadm
 Reviewed by: Mike Gerdts <mike.gerdts@joyent.com> Reviewed by: Dave Eddy
 <dave.eddy@joyent.com> Reviewed by: Josh Wilsdon <josh.wilsdon@joyent.com>
 Approved by: Josh Wilsdon <josh.wilsdon@joyent.com>

---
 src/vm/node_modules/VM.js | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/vm/node_modules/VM.js b/src/vm/node_modules/VM.js
index a475ba9f..5ee8e9a1 100644
--- a/src/vm/node_modules/VM.js
+++ b/src/vm/node_modules/VM.js
@@ -6243,6 +6243,7 @@ function buildZonecfgUpdate(vmobj, payload, log)
         }
         setAttr('com1', 'com1');
         setAttr('com2', 'com2');
+        setAttr('bootrom', 'bootrom');
 
         setAttr('bhyve-extra-opts', 'bhyve_extra_opts');
 
-- 
2.21.0

